From 07cb705ff777da5ec147eb7d2fa0cc4657e6762b Mon Sep 17 00:00:00 2001
From: Nick Kralevich <nnk@google.com>
Date: Tue, 27 Aug 2019 09:48:01 -0700
Subject: [PATCH] sqlite3_android.cpp: disable _TOKENIZE

Comment out the tokenize function. This code doesn't appear to be used.
A future change will further clean up this code and delete it properly.

Bug: 139186193
Test: compiles and boots
Change-Id: I0b2c37b6716162228205fc1ca8bea0f397f36baf
(cherry picked from commit c52a17358593062a7eb75a023c115df9ad89563b)
(cherry picked from commit 2f967a222a948027bb7f02970370ccaa8ae608a2)
---
 android/sqlite3_android.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/android/sqlite3_android.cpp b/android/sqlite3_android.cpp
index b836952..2aee08d 100644
--- a/android/sqlite3_android.cpp
+++ b/android/sqlite3_android.cpp
@@ -226,6 +226,7 @@ struct SqliteUserData {
     UCollator* collator;
 };
 
+#if 0
 /**
  * This function is invoked as:
  *
@@ -402,6 +403,7 @@ static void tokenize(sqlite3_context * context, int argc, sqlite3_value ** argv)
     } while ((token = u_strtok_r(NULL, delim, &state)) != NULL);
     sqlite3_result_int(context, numTokens);
 }
+#endif
 
 static void localized_collator_dtor(UCollator* collator)
 {
@@ -445,6 +447,7 @@ extern "C" int register_localized_collators(sqlite3* handle, const char* systemL
         return err;
     }
 
+#if 0
     // Register the _TOKENIZE function
     err = sqlite3_create_function(handle, "_TOKENIZE", 4, SQLITE_UTF16, collator, tokenize, NULL, NULL);
     if (err != SQLITE_OK) {
@@ -458,6 +461,7 @@ extern "C" int register_localized_collators(sqlite3* handle, const char* systemL
     if (err != SQLITE_OK) {
         return err;
     }
+#endif
 
 
     //// PHONEBOOK_COLLATOR
-- 
2.17.1

